gym_roboy/train_parallel.py

```diff
@@ -21,10 +26,9 @@ def our_env_constructor() -> gym.Env:
 
 num_cpu = int(sys.argv[1])
 env = SubprocVecEnv([setup_constructor(i + 1) for i in range(num_cpu)])
-agent = PPO2("MlpPolicy", env, tensorboard_log=RESULTS_DIR)
+more_exploration = 0.1
+agent = PPO2("MlpPolicy", env, tensorboard_log=TENSORBOARD_DIR, ent_coef=more_exploration)
 
 while True:
-    agent.learn(total_timesteps=1000000)
+    agent.learn(total_timesteps=TRAINING_STEPS_BETWEEN_BACKUPS)
     agent.save(MODEL_FILE)
```

Comment:
I have still the opinion that we should indicate how many training this file (model.pkl) has experienced.

Reply:
Decided with Alex on creating a file naming scheme in a new PR.